exareme2/worker/worker_info/sqlite.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
+from typing import List
+
+from exareme2.worker import config as worker_config
+
+
+def execute_and_fetchall(query) -> List:
```

Comment:
I think it's a bit misleading to have the `execute_and_fetchall`. You don't know from which database you are reading from...

We could name it `sqlite_execute_and_fetchall` but we don't want to bind it to sqlite probably... WDYT? Any better names?

Also, since monetdb is actually very specific and we can't just move to something else, we could rename the ` db_execute_and_fetchall` to `monetdb_execute_and_fetchall`.

Let's discuss this tomorrow.